projects/health-warden.json

```diff
@@ -15,48 +15,43 @@
+    "HealthWarden je mobilní aplikace pro pacienty a webový portál pro doktory pro komplexní vzdálenou péči. Je od základu vyvíjen tak, aby se dokázal přizpůsobit možnostem pacienta, potřebám lékaře i specifikám konkrétní diagnózy. Healthwarden je nástroj na vzdálenou péči o pacienty, co se přizpůsobí životu.",
+    "Lékaři používáním Healthwardenu získají stoprocentní kontrolu nad obsahem vzdálené péče ale díky automatizacím to pro ně nebude představovat žádnou zátěž navíc. HealthWarden obsahuje přizpůsobitelné šablony a automatizované procesy pro rychlé a jednoduché zavedení pacienta. Zároveň nabízí flexibilní integrace do stávajících informačních systémů tak, aby mohl lékař dostávat informace tam, kde je na to zvyklý.",
+    "Díky Healthwardenu může lékař kontinuálně sledovat vitalitu pacienta a to před i po intervenci. Může nastavit virtuální vizitu a upravovat nejen její obsah ale i frekvenci a meze kritických odpovědí. Zároveň můžou být pacientovi pravidelně připomínána lékařská doporučení či medikace."
   ],
   "desc_pieces_en": [
-    "HealthWarden is a mobile and desktop app for complex outpatient care. The average cost of hospitalization per day exceeds $2500, worsening access to health care. Hospital overcrowding blocks other planned or even acute care interventions that could be provided. Doctors face a difficult decision every day: keep the patient in the hospital to make sure everything goes well, or send the patient home to free up hospital capacity?",
-    "HealthWarden app shortens hospitalization time and provides smooth and safe contact between the doctor and the patient during outpatient treatment. It performs regular patient’s condition and health checks using state-of-the-art predictive AI algorithms. Any potential problems are reported to the doctor using anomaly detection tools. Each patient gets structured personalized care based on individual requirements, whereas doctors are provided with well-structured and easily accessible data on each patient as well as their cohort."
+    "HealthWarden is a mobile application for patients and a web portal for doctors for comprehensive remote care. It is developed to be able to adapt to the patient's capabilities, the doctor's needs and the specifics of a particular diagnosis. Healthwarden is a tool for remote patient care that adapts to life.",
+    "By using Healthwarden, doctors gain 100% control over the content of remote care while adding no burden thanks to automation. HealthWarden contains customizable templates and automated processes for quick and easy patient onboarding. It also offers flexible integration into existing information systems so that the doctors can receive information where they are used to it.",
```

Comment:
```suggestion
    "By using HealthWarden, doctors gain 100% control over the content of remote care while adding no burden thanks to automation. HealthWarden contains customizable templates and automated processes for quick and easy patient onboarding. It also offers flexible integration into existing information systems so that the doctors can receive information where they are used to it.",
```